sys-libs/mesa/mesa-17.1.7.recipe

```diff
@@ -27,119 +28,106 @@ REQUIRES="
 PROVIDES_devel="
 	mesa${secondaryArchSuffix}_devel = $portVersion
 	devel:libGL$secondaryArchSuffix = $portVersion
-	devel:libEGL$secondaryArchSuffix = $portVersion
+	devel:libEGL$secondaryArchSuffix = 1.0.0
 	"
 REQUIRES_devel="
 	mesa$secondaryArchSuffix == $portVersion base
 	"
 
+SUMMARY_swpipe="The Mesa LLVM enhanced Gallium software pipe renderer"
+DESCRIPTION_swpipe="This 3D BGLRenderer add-on provides Gallium LLVM \
+enhanced software rendering. Software pipe rendering performs all \
+3D rendering on the systems CPU and doesn't require any specialized \
+hardware. The usage of LLVM over traditional rasterization gives this \
+renderer a boost in performance.
+
+Gallium software pipe rendering is in an extremely early state as of \
+this version of Mesa, and may not function as expected."
+
+PROVIDES_swpipe="
+	mesa${secondaryArchSuffix}_swpipe = $portVersion
+	"
+REQUIRED_swpipe="
+	mesa$secondaryArchSuffix == $portVersion base
+	lib:libLLVM_4.0$secondaryArchSuffix
+	"
+
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libz$secondaryArchSuffix
+	llvm${secondaryArchSuffix}
```

Comment:
This doesn't seem correct.  llvm is a static library used during linkage. I'm 90% sure we don't need llvm installed to leverage the llvmpipe renderer

Reply:
llvm is now built with shared libs, there is a runtime dep. Here the line is just moved from BUILD_PREREQUIRES, because it's a library build dep, not a tool.